packages/enzyme/src/ShallowWrapper.js

```diff
@@ -152,11 +152,17 @@ class ShallowWrapper {
     return this[ROOT];
   }
 
+  getRootNodeInternal() {
```

Comment:
we tried pretty hard in v3 to minimize the public API. Can this be a normal function instead of a class method?

Reply:
I've fixed it.